src/main/java/org/kohsuke/github/GHRepositorySearchBuilder.java

```diff
@@ -47,9 +57,11 @@ public GHRepositorySearchBuilder size(String v) {
      * @param v
      *            the v
      * @return the gh repository search builder
+     * @deprecated use {@link #fork(GHFork)} instead.
      */
+    @Deprecated
     public GHRepositorySearchBuilder forks(String v) {
-        return q("forks:" + v);
```

Comment:
Well, this is was a bug.

Reply:
Nice pickup! Thanks for reviewing this PR 🎉